src/devices_models/devices/common/objective_functions.jl

```diff
@@ -756,8 +756,12 @@ function _add_pwl_term!(
     ::U,
     ::V,
 ) where {T <: PSY.ThermalGen, U <: VariableType, V <: ThermalDispatchNoMin}
+    multiplier = objective_function_multiplier(U(), V())
```

Comment:
Was this an oversight?

Reply:
yeah it is used down below, but was never defined in this function. There were several variables in this situation.

Reply:
Thanks for making these fixes, I completely forgot about them after I made the changes to SIIPExamples